_posts/2017-09-29-make-connections.html

```diff
@@ -9,6 +9,8 @@
 ---
 <p>FT Labs is preparing to launch a new game on Google Home. Here is a glimpse of what is to come, and some learnings acquired in the process of developing for voice devices.</p>
 
+<p>Update: as <a href="https://twitter.com/FTLabs/status/932982974030761985" target="_blank">tweeted</a>, the game is live on Google Home and <a href="https://assistant.google.com/services/a/id/5c19766e2cd9d4c3/" target="_blank">Google Assistant</a>. Invoked by saying, "OK Google. Talk to Make Connections".</p>
```

Comment:
Not sure the link to the tweet brings anything. Either remove or embed the tweet into the post, rather than link out.